docs/sources/operations/storage/filesystem.md

```diff
@@ -42,103 +42,3 @@ The durability of the objects is at the mercy of the filesystem itself where oth
 ### High Availability
 
 Running Loki clustered is not possible with the filesystem store unless the filesystem is shared in some fashion (NFS for example).  However using shared filesystems is likely going to be a bad experience with Loki just as it is for almost every other application.
-
```

Comment:
Is that a not working anymore ? just curious.

Reply:
I'm not sure if it works we haven't had time to test it, the configs for it were complex and I don't know that we are ready or even really want to support this so I'm removing the references to it in the docs. I will add a comment in the upgrade guide